### ISSUE TITLE
Can't find Phantomjs

### ISSUE THREAD

**Issue by RobinClowers** [keathley/wallaby#327]
I just upgraded from 0.15.0 to 0.19.1, and after updating my tests to match the new api, wallaby fails to start on my CI server with this error:

```
Wallaby can't find phantomjs. Make sure you have phantomjs installed
and included in your path, or that your `config :wallaby, :phantomjs`
setting points to a valid phantomjs executable.
```

We are using phantom-prebuilt, here is the config for wallaby:

```elixir
config :wallaby,
  base_url: "http://#{dev_host}:#{dev_port}",
  screenshot_on_failure: true,
  phantomjs: "node_modules/.bin/phantomjs",
  phantomjs_args: "--webdriver-logfile=log/phantomjs.log",
  js_logger: "log/acceptance_browser.log"
```

This is running on Elixir 1.5 with Erlang 20.0.

**Comment by aaronrenner** [comment]
@RobinClowers I see what you mean. It turns out that `System.find_executable/1` doesn't like relative paths. I think there's a few options for a workaround.

1. Update your config to use `Path.expand/1`
    ```elixir
    config :wallaby,
      phantomjs: Path.expand("node_modules/.bin/phantomjs", ".")
    ```
2. Add phantomjs to your PATH before running the command and use the following config:
    ```elixir
    config :wallaby,
      phantomjs: "phantomjs" # Or just use the default config
    ```
3. Use an absolute path:
    ```elixir
    config :wallaby,
      phantomjs: "/home/user/node_modules/.bin/phantomjs"
    ```
Thanks for pointing this out and it seems like this warrants a code or documentation fix.

**Comment by aaronrenner** [comment]
@RobinClowers I just tried it out and this issue has been fixed on master (#296).
